entries/a/acorns.com.json

```
@@ -3,14 +3,16 @@
     "domain": "acorns.com",
     "url": "https://www.acorns.com/",
     "tfa": [
-      "sms"
+      "sms",
+      "email"
     ],
     "documentation": "https://www.acorns.com/support/how-do-i-turn-on-two-factor-authentication-on-my-account/",
+    "notes": "You will need to verify one piece of contact informaiton and using your PIN or Face ID in-app will not activate 2FA.",
```

```suggestion
    "notes": "You will need to verify one piece of contact information before activating 2FA",
```

Isn't the second part (the in-app security) unnecessary? I would argue that most know that it doesn't activate 2FA.

---

They are saying that if you use those forms of authentication they will not ask you for your second factor.

---

So maybe better to say: `You will need to verify email or phone in order to activate 2FA. If you use biometrics to unlock the app it will not prompt you for your second factor.`

---

I would remove the part about the app, but otherwise, looks good to me.